CMSIS/NN/Include/arm_nnsupportfunctions.h

```diff
@@ -633,6 +633,36 @@ __STATIC_FORCEINLINE q31_t arm_nn_requantize(const q31_t val, const q31_t multip
                                        RIGHT_SHIFT(shift));
 }
 
+/**
```

Comment:
Why not use the DSP/Support/arm_copy_q7 ?
The DSP version is not using predication. Not sure it will make a big difference (or perhaps only for small buffers).

Reply:
@christophe0606 Ah.. didn't know that there was a version there! I had tried a similar non-predicated one the version that you see here. There were some minor differences in the performance, but the major difference was in in-lining. It is replaced by 4 instructions. Nice and compact.  If arm_copy_q7() can be made as an inline function, I am happy to use that.

Reply:
@felix-johnny  Ok but it will have to wait for the big rework of arm_math.h